tests/pages/books/book/pages.js

```diff
@@ -19,6 +19,10 @@ export default create({
     return find(this.textArea).style.fontFamily;
   },
 
+  toggleHover() {
+    return document.querySelector('.bookShelf').children[1].classList.remove('editHover');
```

Comment:
Children should be defined as param. A method does not tell that it hovers the first one. Also, you can mix it with "pane" method to get the children easier (for instance it could be `toggleHover(pane)` where pane is the result from `pageItem(index)` call